webapp/sources/rudder/rudder-core/src/main/scala/com/normation/rudder/ncf/TechniqueWriter.scala

```diff
@@ -330,9 +350,12 @@ class TechniqueWriter (
     }
   }
 
+  def techniquePath(technique : Technique) = s"techniques/${technique.category}/${technique.bundleName.value}/${technique.version.value}"
```

Comment:
What happens with category if it's a sub category? Does is contains the full relative path?
Also, I think we are going to have problem by calling `Technique` both `com.normation.rudder.ncf.Technique` and `com.normation.cfclerk.domain.Technique`. Can we rename the first one `EditorTechnique` ? (or something letting us know that it's not the same).